### PR TITLE
ADDED: Authentication Pages

### PR DESCRIPTION
**Pull request by AKSHAYAPALCHAMY-P**

### CODE REVIEW

File: LibraryManagementSystem.Common/TableObjects/BookDto.cs [hunk 1]
```diff
@@ -0,0 +1,10 @@
+﻿namespace LibraryManagementSystem.Common
+{
+    public class BookDto
+    {
+        public string BookNo {  get; set; }
```
Comment (ManiRaj-021199) [inline LibraryManagementSystem.Common/TableObjects/BookDto.cs]: Change the type as integer (remove if you don't need).
Change the name as Id

File: LibraryManagementSystem.Common/TableObjects/BookDto.cs [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+        public string NameOfTheBook {  get; set; }
```
Comment (ManiRaj-021199) [inline LibraryManagementSystem.Common/TableObjects/BookDto.cs]: Change as Name

File: LibraryManagementSystem.Common/TableObjects/BookDto.cs [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+        public string AuthorName {  get; set; }
```
Comment (ManiRaj-021199) [inline LibraryManagementSystem.Common/TableObjects/BookDto.cs]: Change as Author

File: LibraryManagementSystem.Data/TableObjects/Book.cs [hunk 4]
```diff
@@ -0,0 +1,22 @@
+﻿using LibraryManagementSystem.Common;
+
+namespace LibraryManagementSystem.Data
+{
+    public class Book
+    {
+        public int Id { get; set; }
+        public string BookNo { get; set; }
```
Comment (ManiRaj-021199) [inline LibraryManagementSystem.Data/TableObjects/Book.cs]: What is BookNo?

File: LibraryManagementSystem.Data/TableObjects/Book.cs [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+        public string NameOfTheBook { get; set; }
```
Comment (ManiRaj-021199) [inline LibraryManagementSystem.Data/TableObjects/Book.cs]: Name

File: LibraryManagementSystem.Data/TableObjects/Book.cs [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+        public string AuthorName { get; set; }
```
Comment (ManiRaj-021199) [inline LibraryManagementSystem.Data/TableObjects/Book.cs]: Author

**Review by ManiRaj-021199 — CHANGES_REQUESTED** [review]
Please check the comments.

File: LibraryManagementSystem.Common/TableObjects/BookDto.cs (hunk 1 above)
Comment (AKSHAYAPALCHAMY-P) [reply LibraryManagementSystem.Common/TableObjects/BookDto.cs]: Done

File: LibraryManagementSystem.Common/TableObjects/BookDto.cs (hunk 2 above)
Comment (AKSHAYAPALCHAMY-P) [reply LibraryManagementSystem.Common/TableObjects/BookDto.cs]: Done

File: LibraryManagementSystem.Common/TableObjects/BookDto.cs (hunk 3 above)
Comment (AKSHAYAPALCHAMY-P) [reply LibraryManagementSystem.Common/TableObjects/BookDto.cs]: Done

File: LibraryManagementSystem.Data/TableObjects/Book.cs (hunk 4 above)
Comment (AKSHAYAPALCHAMY-P) [reply LibraryManagementSystem.Data/TableObjects/Book.cs]: Done

File: LibraryManagementSystem.Data/TableObjects/Book.cs (hunk 5 above)
Comment (AKSHAYAPALCHAMY-P) [reply LibraryManagementSystem.Data/TableObjects/Book.cs]: Done

File: LibraryManagementSystem.Data/TableObjects/Book.cs (hunk 6 above)
Comment (AKSHAYAPALCHAMY-P) [reply LibraryManagementSystem.Data/TableObjects/Book.cs]: Done

**Review by ManiRaj-021199 — APPROVED** [review]

**Pull request merged**